### DISCUSSION TITLE
Tailwind v2.1.2 not working on windows got error 'TAILWIND_MODE' is not recognized as an internal or external command

### DISCUSSION THREAD

**Discussion by arkangel-app** [tailwindlabs/tailwindcss#4334] [Help]
In Ubuntu is everything ok. The project runs. But when I am trying to install in windows I got the error:
'TAILWIND_MODE' is not recognized as an internal or external command
I think is something with the watcher. Anybody had same issue before?

My package.js:
>    "scripts": {
        "storybook": "TAILWIND_MODE=watch start-storybook -p 6006"
     ....}

**Comment by jenstornell** [comment]
I hade the same problem and I didn't figure it out. Instead I found a great video tutorial to install it.

https://www.youtube.com/watch?v=lYWQdxHGnkc&feature=youtu.be

**Comment by dance2die** [comment]
The issue is how environment variables are set differently in Linux and Windows.

Your `storybook` script works in Linux because that's how you set variables in common linux shells, such as bash, zsh.
Windows require a `set` variable, and need to chain with `&&`.

You can find the same issue here https://stackoverflow.com/questions/40806796/setting-environment-variables-package-json-in-windows-10.

You can get around the issue with [cross-env](https://www.npmjs.com/package/cross-env) to set the variable that works across platforms.
(the workaround is also in that stackoverflow).
